Directory.Build.props

```diff
@@ -9,9 +9,7 @@
 
   <PropertyGroup>
     <BuildArchitecture>$([System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture.ToString().ToLowerInvariant())</BuildArchitecture>
-    <Architecture Condition="'$(Architecture)' == '' AND '$(BuildArchitecture)' == 'arm64'">$(BuildArchitecture)</Architecture>
-    <Architecture Condition="'$(Architecture)' == '' AND '$(BuildArchitecture)' == 's390x'">$(BuildArchitecture)</Architecture>
-    <Architecture Condition="'$(Architecture)' == ''">x64</Architecture>
+    <Architecture Condition="'$(Architecture)' == ''">$(BuildArchitecture)</Architecture>
```

Comment:
FYI, these sort of changes break developing using Visual Studio - where an x86 build of Visual Studio is used to create the x64 SDK of .NET. I made similar fixes to some .NET repos and then had to revert them.

Reply:
Copy that, reverted.

Reply:
@omajid I pushed an attempt at a better implementation. Checks seem to be happy, what do you think?